sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/AssignDestinations.java

```diff
@@ -41,23 +45,30 @@ public AssignDestinations(DynamicDestinations dynamicDestinations) {
   @Override
   public PCollection<Row> expand(PCollection<Row> input) {
 
-    final Schema inputSchema = input.getSchema();
     final Schema outputSchema =
         Schema.builder()
-            .addRowField("data", inputSchema)
-            .addRowField("dest", dynamicDestinations.getMetadataSchema())
+            .addStringField("dest")
```

Comment:
These field names should go to consts somewhere (preferably in a proto).

Reply:
Sure will create constants. 

Do we need protos here? This code is just in Java